recurly.go

```diff
@@ -234,12 +234,13 @@ func newResponse(r *http.Response) *response {
 }
 
 func (r *response) populatePageCursor() {
-	links, ok := r.Response.Header["Link"]
-	if !ok || len(links) == 0 {
+	links := r.Response.Header.Get("Link")
+
+	if len(links) == 0 {
```

Comment:
to keep consistent with the library's styling, please move this up a line and use `if links != "" {`